homeassistant/components/skybell/binary_sensor.py

```diff
@@ -16,11 +19,21 @@
 
 SCAN_INTERVAL = timedelta(seconds=10)
 
-# Sensor types: Name, device_class, event
-SENSOR_TYPES = {
-    "button": ["Button", DEVICE_CLASS_OCCUPANCY, "device:sensor:button"],
-    "motion": ["Motion", DEVICE_CLASS_MOTION, "device:sensor:motion"],
-}
+
+class SensorTypeClass(NamedTuple):
+    """Metadata for sensor types."""
+
+    name: str
+    # Sensor types: Name, device_class, event
```

Comment:
In essence, this comment should become unnecessary.

Reply:
Gone.